File: src/typegraph/specs/codegen/rpc/python/client_mock.py
# Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
# SPDX-License-Identifier: MPL-2.0

import json

from client import rpc_request


first = rpc_request("hello", {"name": "world"})
second = rpc_request("foo")

print(json.dumps({"first": first, "second": second}))

File: src/typegraph/specs/codegen/rpc/typescript/client_mock.ts
// Copyright Metatype OÜ, licensed under the Mozilla Public License Version 2.0.
// SPDX-License-Identifier: MPL-2.0

import { rpcRequest } from "./client.ts";

const first = rpcRequest("hello", { name: "world" });
const second = rpcRequest("foo");

console.log(JSON.stringify({ first, second }));
